### DISCUSSION TITLE
Shipping PSR-6 cache store

### DISCUSSION THREAD

**Discussion by ostrolucky** [spatie/guzzle-rate-limiter-middleware#26] [Ideas]
Hello, how about shipping PSR-6 store with this package? It could even be optional dependency (but even if it was added to `require`, it's very minimal package). Building and maintaing store for it would be pretty easy and is quite universal.

**Comment by freekmurze** [comment]
If this doesn't require too many changes in the package, I'd accept a PR that adds this.
